### PR TITLE
Add icingacli-director check to ITL

### PR DESCRIPTION
**Pull request by widhalmt**
Including documentation.
Add a hint to the documentation of all icingacli commands that Icinga 2 has to be allowed access to the icingaweb2 directory.

fixes: #5920

### CODE REVIEW

File: doc/10-icinga-template-library.md [hunk 1]
```diff
@@ -2519,6 +2521,16 @@ icingacli_businessprocess_config          | **Optional.** Configuration file con
...
+#### Director <a id="plugin-contrib-icingacli-director"></a>
+
+This subcommand is provided by the [director module](https://github.com/Icinga/icingaweb2-module-director) > 1.4.2 and executed as `icingacli director health check`. Please refer to the [documentation](https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/60-CLI.md#health-check-plugin) for all available sub-checks.
+
+Custom attributes passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                                      | Description
+------------------------------------------|-----------------------------------------------------------------------------------------
+icingacli_director_check                  | **Optional** Run only a specific test suite
+icingacli_director_db                     | **Optional** Use a specific Icinga Web DB resource
```
Comment (mcktr) [inline doc/10-icinga-template-library.md]: Just a very little cosmic thing: The punctuation marks are missing :)

**Comment by widhalmt** [comment]
Thanks for pointing out. I added the punctuation marks by amending. Is there anything else that needs to be changed?

File: doc/10-icinga-template-library.md (hunk 1 above)
Comment (widhalmt) [reply doc/10-icinga-template-library.md]: I was quite sure I double-checked before committing. Meh. Thanks for finding the missing marks.

File: itl/plugins-contrib.d/icingacli.conf [hunk 2]
```diff
@@ -51,3 +51,20 @@ object CheckCommand "icingacli-businessprocess" {
...
+
+        command += [ "director", "health", "check" ]
+
+        arguments = {
+                "--check" = {
+                        value = "$icingacli_director_check$"
+                        description = "Run only a specific test suite"
+                }
+                "--db" = {
+                        set_if = "$icingacli_director_db$"
```
Comment (lazyfrosch) [inline itl/plugins-contrib.d/icingacli.conf]: Shouldn't this be a `value`?

Comment (widhalmt) [reply itl/plugins-contrib.d/icingacli.conf]: You're totally right. Fixed it. Sorry.

**Comment by lazyfrosch** [comment]
Awesome!

**Comment by widhalmt** [comment]
Thanks for merging.

**Pull request merged**